content/en/dynamic_instrumentation/symdb/java.md

```diff
@@ -70,7 +65,26 @@ java \
    $ java -jar my-service.jar -javaagent:dd-java-agent.jar ...
    ```
 
-2. After starting your service with Symbol Database enabled, you can use Symbol Database's IDE-like features on the [APM > Dynamic Instrumentation page][4].
+3. After starting your service with Dynamic Instrumentation with autocomplete and search enabled, you can use Dynamic Instrumentation's IDE-like features on the [APM > Dynamic Instrumentation page][4].
+
+## Additional configuration
+
+### Third party detection
+
+If your code is accidentally recognized as third-party code or third-party code is accidentally recognized as user code, you can configure the third-party detection with these settings:
+
+```
+export DD_THIRD_PARTY_DETECTION_EXCLUDES=<LIST_OF_USER_CODE_PACKAGE_PREFIXES>
+export DD_THIRD_PARTY_DETECTION_INCLUDES=<LIST_OF_ADDITIONAL_THIRD_PARTY_PACKAGE_PREFIXES>
+```
+
+or:
+
+```
+  -Ddd.third.party.detection.excludes=<LIST_OF_USER_CODE_PACKAGE_PREFIXES> \
```

Comment:
no `detection` in config options:
https://github.com/DataDog/dd-trace-java/blob/d62398b6a2f36fcb51b66023ac3a831fce1714ba/dd-trace-api/src/main/java/datadog/trace/api/config/DebuggerConfig.java#L40

Reply:
Damn, now it is inconsistent across libraries 😭 My bad.